packages/flutter_tools/test/general.shard/flutter_platform_test.dart

```diff
@@ -122,4 +122,37 @@ void main() {
       expect(flutterPlatform.uriConverter?.call('hello'), 'hello/test');
     });
   });
+
+  group('generateTestBootstrap', () {
+    test('with null packageConfigUri', () {
+      final String contents = generateTestBootstrap(
+        testUrl: Uri.parse('file:///Users/me/some_package/test/some_test.dart'),
+        host: InternetAddress('127.0.0.1', type: InternetAddressType.IPv4),
+      );
+      // IMPORTANT: DO NOT RENAME, REMOVE, OR MODIFY THE 'packageConfigLocation'
```

Comment:
rather than an `IMPORTANT` comment, maybe explicitly denote in the test name we're checking for the constant we rely on, e.g.

"writes a `const packageConfigLocation` string to the test bootstrap"